crates/sui/src/client_commands.rs

```diff
@@ -622,6 +622,10 @@ pub enum SuiClientCommands {
         /// Log extra gas-related information
         #[arg(long, short)]
```

Comment:
Do you need the short flag? The issue is that due to many commands, often if two commands that start with the same letter have the `short` derive property, it will lead to conflicts and errors. Maybe it's better to keep it consistent and only have the long property.